app/src/main/java/org/eyeseetea/malariacare/data/database/model/SurveyDB.java

```diff
@@ -1055,6 +1060,37 @@ public static void deleteAll() {
         }
     }
 
+    public static void deleteOlderSentSurveys() {
+
+        List<SurveyDB> sentSurveys = getAllSentSurveys();
+
+        for (SurveyDB surveyDB : sentSurveys) {
+
+            if (isSurveyOlderThan(surveyDB, MaxDaysBeforeDeletingSentSurveys)) {
+
+                new Delete().from(ValueDB.class).where(
+                        ValueDB_Table.id_survey_fk.eq(surveyDB.getId_survey()));
+
+                surveyDB.delete();
```

Comment:
@Amejia481 please instead of getting all surveys and looping on all of then manually matching those that are older than the given date, please create directly a DBFlow call returning only those surveys older than a given time. Otherwise we would be introducing a performance problem.

Reply:
I was trying to find a way of applying this  sql statement with dbflow 
~~~~
Where event_date < date('now','-14 years')
~~~~

But I was unable to do the same with DBFlow, I will do some research.